sdk/typescript/introspector/test/invoke.spec.ts

```diff
@@ -262,17 +262,68 @@ describe("Invoke typescript function", function () {
 
       const scanResult = scan(files)
 
-      // Mocking the fetch from the dagger API
-      const input = {
-        parentName: "HelloWorld", // HelloWorld
-        fnName: "greet", // helloWorld
-        parentArgs: {},
-        fnArgs: { name: "Dagger" },
-      }
+      // We wrap the execution into a Dagger connection
+      await connection(async () => {
+        const constructorInput = {
+          parentName: "HelloWorld", // HelloWorld
+          fnName: "", // call constructor
+          parentArgs: {
+            prefix: "test",
```

Comment:
Shouldn't this use a different value from the default in `alias/index.ts`?

Reply:
Nope, the constructor does not have the argument to override the prefix, I just add the state for dagger